README.md

```diff
@@ -0,0 +1,13 @@
+This repository has AWS lambda functions for the economics APIs.
+
+## Available APIs
+
+- **Status:** https://api.economics.hathor.network/status
```

Comment:
Since we won't have the cache, should we deploy this API?

Reply:
I agree, there is actually no need for that right now. If we decide not to have this API, should we update the design? Maybe add a note to explain why one endpoint is missing?

@msbrogli @obiyankenobi

Reply:
Yeah, we should explain on the design.

Reply:
I think it is important for debugging. I think we should keep it.

Reply:
Ok, we can keep it.